Purchases/Purchasing/StoreKit2/StoreKit2TransactionListener.swift

```diff
@@ -69,20 +71,21 @@ class StoreKit2TransactionListener {
 @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
 private extension StoreKit2TransactionListener {
 
-    func handle(transactionResult: VerificationResult<StoreKit.Transaction>) async {
+    /// - Returns: whether the transaction was verified
+    func handle(transactionResult: VerificationResult<StoreKit.Transaction>) async -> Bool {
         switch transactionResult {
-        case .unverified(let unverifiedTransaction, let verificationError):
-            // todo: update once StoreKit fixes the issue with verifying sandbox purchases.
-            #if DEBUG
-            Logger.debug("The transaction has failed verification, but Sandbox purchases dont' support" +
```

Comment:
I verified this works in a device now on Sandbox.

Reply:
🙌 finally!